ppx/test/input_errors_01_element_attribute.ml

```diff
@@ -1,3 +0,0 @@
-[@@@react.dom]
-
-let%component make () = div ~lola:1 ()
```

Comment:
This test does not make any sense anymore, as the ppx does not check that the prop name is valid.